### ISSUE TITLE
Query data error when selecting column of type  ARRAY<STRING>

### ISSUE THREAD

**Issue by chrisleavoy** [grafana/google-bigquery-datasource#105]
I have a table with a column of type `ARRAY<STRING>`.  The BigQuery plugin returns a 500 Query data error when trying to select the column or use the `select *`. The same behaviour occurs when using the query builder or code editor.

```
 logger=context t=2022-04-20T16:26:16.49+0000 lvl=eror msg="Query data error" error="failed to query data: Failed to query data: rpc error: code = Unavailable desc = error reading from server: EOF" remote_addr=127.0.0.1
 logger=context t=2022-04-20T16:26:16.49+0000 lvl=eror msg="Request Completed" method=POST path=/api/ds/query status=500 remote_addr=127.0.0.1 time_ms=1858 size=30 ...
```

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/1428245/164277913-62845fd7-e719-4fec-8ec6-e2d1161f3104.png">

**Comment by dprokop** [comment]
Thanks, @chrisleavoy - we only added support for Records, but the Arrays support is coming with #108

**Comment by dprokop** [comment]
@chrisleavoy this is now fixed in 1.0.3 - let us know if there are any more issues related to the ARRAY type.
